### ISSUE TITLE
Unit tests not working

### ISSUE THREAD

**Issue by Gargamella** [steveblue/angular2-rollup#7]
```
PhantomJS 2.1.1 (Mac OS X 0.0.0) ERROR
  ReferenceError: Can't find variable: require
  at karma-shim.js:3
PhantomJS 2.1.1 (Mac OS X 0.0.0) ERROR
  ReferenceError: Can't find variable: require
  at karma-shim.js:3
```

**Comment by steveblue** [comment]
@Gargamella If you can't wait, add require to the devDependencies in package.json and this issue should be resolved. I will add it back in if I get some time later this evening.

**Comment by Gargamella** [comment]
Thank you, but still not working
```
Gargamella@MBP-di-Gargamella:~/Sources/Github/angular2-rollup$ npm i --save-dev require
angular2-rollup@2.0.1 /Users/Gargamella/Sources/Github/angular2-rollup
└─┬ require@2.4.20 
  ├── std@0.1.40 
  └─┬ uglify-js@2.3.0 
    ├── async@0.2.10 
    ├── optimist@0.3.7 
    └── source-map@0.1.43 

Gargamella@MBP-di-Gargamella:~/Sources/Github/angular2-rollup$ npm test

> angular2-rollup@2.0.1 pretest /Users/Gargamella/Sources/Github/angular2-rollup
> npm run lint


> angular2-rollup@2.0.1 lint /Users/Gargamella/Sources/Github/angular2-rollup
> tslint --force "src/app/**/*.ts"

src/app/app.component.ts[5, 15]: The selector of the component "AppComponent" should have prefix "my" (https://goo.gl/cix8BY)

> angular2-rollup@2.0.1 test /Users/Gargamella/Sources/Github/angular2-rollup
> karma start


START:
24 10 2016 21:43:45.269:INFO [karma]: Karma v1.3.0 server started at http://localhost:9876/
24 10 2016 21:43:45.275:INFO [launcher]: Launching browser PhantomJS with unlimited concurrency
24 10 2016 21:43:45.286:INFO [launcher]: Starting browser PhantomJS
24 10 2016 21:43:45.827:INFO [PhantomJS 2.1.1 (Mac OS X 0.0.0)]: Connected on socket /#rmX24Ug9dRQFCbftAAAA with id 99576140
PhantomJS 2.1.1 (Mac OS X 0.0.0) ERROR
  ReferenceError: Can't find variable: require
  at karma-shim.js:3
PhantomJS 2.1.1 (Mac OS X 0.0.0) ERROR
  ReferenceError: Can't find variable: require
  at karma-shim.js:3

Finished in 0.033 secs / 0 secs

SUMMARY:
✔ 0 tests completed
----------|----------|----------|----------|----------|----------------|
File      |  % Stmts | % Branch |  % Funcs |  % Lines |Uncovered Lines |
----------|----------|----------|----------|----------|----------------|
----------|----------|----------|----------|----------|----------------|
All files |      100 |      100 |      100 |      100 |                |
----------|----------|----------|----------|----------|----------------|

npm ERR! Test failed.  See above for more details.

```

**Comment by steveblue** [comment]
Yes, the problem probably stems from porting this from a webpack implementation. I will need a lot more time to diagnose. If anyone wants to contribute a fix, I'll merge it in.

**Comment by steveblue** [comment]
@Gargamella Please verify the tests are fixed in the latest release `2.0.2-r.1`. I will be versioning up the repo with the latest versions of Angular 2 soon.

**Comment by steveblue** [comment]
Verified tests are fixed up to latest version `2.1.2`
